src/views/welcome/components/WelcomeButtons.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+
+import { Button, ButtonVariant, StackItem } from '@patternfly/react-core';
+
+import { welcomeModalButtons } from '../utils/constants';
+import { WelcomeModalButtonsData } from '../utils/types';
+
+type WelcomeButtonsProps = {
+  onClose: () => Promise<void> | void;
+};
+
+const WelcomeButtons: FC<WelcomeButtonsProps> = ({ onClose }) => {
+  const history = useHistory();
+  const { ns } = useParams<{ ns: string }>();
+
+  return (
+    <>
+      {welcomeModalButtons(ns).map((modalButton: WelcomeModalButtonsData, index: React.Key) => (
+        <StackItem key={index}>
```

Comment:
using the index as a key is not recommended and should be avoided, you can use `modalButton.name` as unique key in this case.
If you want to read more about why this should be avoided you can go to react docs here and keep reading until the "Pitfall" yellow part below: https://react.dev/learn/rendering-lists#where-to-get-your-key

Reply:
Yeah, I know. Just sometimes it is hard to see what else I can use instead of index...